eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -191,7 +191,6 @@ stages:
                       - template: /eng/common/pipelines/templates/steps/publish-blobs.yml
                         parameters:
                           FolderForUpload: '$(Pipeline.Workspace)/${{parameters.DocArtifact}}/${{artifact.name}}'
-                          BlobSASKey: '$(azure-sdk-docs-prod-sas)'
```

Comment:
Thanks for cleaning these up. Please be sure to delete the value in KV and removed it from the variable group as well.